catalyst/dl/scripts/run.py

```diff
@@ -44,18 +44,25 @@ def build_args(parser: ArgumentParser):
         metavar="PATH",
         help="path to latest checkpoint"
     )
+    parser.add_argument(
```

Comment:
Could you replace `parser.add_argument(action="store_true", ...)` with `utils.boolean_flag(...)`?

Reply:
+

Reply:
done